site/recipes/recipe_different_processing_against_stream.md

```diff
@@ -10,27 +10,35 @@ In this instance, we can take the stream of mileage sensor readings and apply mu
 
 ## Setting up the application
 
-We assume that the environment has been set up following the steps outlined in the [Getting Started Guide](../docs/quarks-getting-started). Let's begin by creating a `DirectProvider` and `Topology`. We choose a `DevelopmentProvider` so that we can view the topology graph using the console URL (refer to the [Application Console](../docs/console) page for a more detailed explanation of this provider). The initial mileage value and the number of miles in a typical delivery route have also been defined.
+We assume that the environment has been set up following the steps outlined in the [Getting started guide](../docs/quarks-getting-started). Let's begin by creating a `DirectProvider` and `Topology`. We choose a `DevelopmentProvider` so that we can view the topology graph using the console URL (refer to the [Application console](../docs/console) page for a more detailed explanation of this provider). The gas mileage bounds, initial mileage value, and the number of miles in a typical delivery route have also been defined.
 
 ```java
     import java.text.DecimalFormat;
-    import java.util.Random;
     import java.util.concurrent.TimeUnit;
 
     import com.google.gson.JsonObject;
 
+    import quarks.analytics.sensors.Ranges;
     import quarks.console.server.HttpServer;
     import quarks.providers.development.DevelopmentProvider;
     import quarks.providers.direct.DirectProvider;
+    import quarks.samples.utils.sensor.SimpleSimulatedSensor;
     import quarks.topology.TStream;
     import quarks.topology.Topology;
 
     public class ApplyDifferentProcessingAgainstStream {
         /**
-         * Hypothetical values for the initial gas mileage and the
-         * number of miles in a typical delivery route
+         * Gas mileage value bounds
```

Comment:
~nit: since you're in here improving things, no good deed goes unpunished :-), the first time I looked at this recipe, I was a initially confused because it uses the term "mileage" in variety of places (e.g., stream name, constants) where the values are really "miles per gallon"; the definition of "mileage" means distance.  Consider renaming things.